features/20-exchange-access-token.feature

```diff
@@ -0,0 +1,61 @@
+@smart
+Feature: App exchanges authorization code for access token
+
+    Scenario: Success response has all required parameters
+        Given OAuth is enabled
+        And I am not logged in
+        When I ask for authorization
+        And I exchange my grant code
+        Then the response code should be 200
+        And the JSON response will contain access_token
+        And the JSON response will contain token_type
+        And the JSON response will contain scope
+        And the JSON response will contain state
```

Comment:
I'm going off of http://docs.smarthealthit.org/authorization/#step-3 for this list, but neither the auth-proxy implementation nor any of the vendors provide "state" at this point, so this is likely a departure from the OAuth spec. Can you shed any light on that?